fifa_draft/utils.py

```diff
@@ -42,7 +54,7 @@ def team_form_validation(request, form, profile):
         messages.error(request, "Featured image is too big (max 3mb)")
```

Comment:
Widzę, że poprawiłeś, ale zastanawiam się, czy na pewno tak chciałeś mieć.

Czy nie chciałeś po prostu mieć tak?
```
        elif profile in team.belongs_group.members.all():
            messages.error(request, "You have already team in this group")
    else:
        messages.error(request, "Featured image is too big (max 3mb)")

    return form_valid
```

Wtedy zawsze zwraca boola i domyślam się, że taki był twój cel. A w tym przypadku type hint to będzie sam `bool`.

Reply:
+ jak już tu jestem to polecam zmienne, które są boolami, nazywać poprzez formę pytającą, tzn. `is_form_valid`. Taka konwencja od razu sugeruje, że ta zmienna to bool i True oznacza twierdząco na to pytanie, a false przecząco.